recipes/oras-py/meta.yaml

```diff
@@ -0,0 +1,49 @@
+{% set name = "oras-py" %}
+{% set version = "0.0.11" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/oras-project/oras-py/archive/refs/tags/{{ version }}.tar.gz
+  sha256: 3a1c991e122435cc3ae080fbdb0f7addcb75b17a5ec8afa22f87afc55dc93b7b
+
+build:
+  noarch: python
+  number: 0
+  script: {{ PYTHON }} -m pip install . -vv
+  entry_points:
+    - oras-py=oras.cli:run
+
+requirements:
+  host:
+    - python >=3.7
+    - pip
```

Comment:
Looks like you have a hard requirement on pytest-runner at build time. See https://github.com/oras-project/oras-py/blob/32d64e452869ac5b04ffa2f883ea3bc27c7e8724/setup.py#L86
```suggestion
    - pip
    - pytest-runner
```

Reply:
@ocefpaf that is likely an oversight - if we remove it will that fix the builds here?

Reply:
No idea. We will deal one error at a time.

Reply:
okay I'll remove it and give it a shot then.

Reply:
yeah looks like we need `pytest-runner`, judging from the Azure logs :)